### ISSUE TITLE
Triangulering

### ISSUE THREAD

**Issue by einand** [piratpartiet/www.overvakad.se#1]
Triangulering av mobiltelefoner
Även positionen till äldre mobiltelefoner utan inbyggd GPS kan beräknas genom att mäta signalstyrkan vid minst tre basstationer. Denna typ av mätning är en nödvändig del av mobiltelefonnätets funktion, men positionen lagras också enligt EU-direktivet om datalagring.

Denna texten är helt felformulera, Triangulering krävs två kända punkter, så räknar du ut den tredje. I en "triangel". 

Förutom detta, så är detta fel i huvuddraget också eftersom det är Trilateration som används för att beräkna avståndet. Vilket kan ge en så låg felmarginal som 50 meter.
https://sv.wikipedia.org/wiki/Trilateration

**Comment by redhog** [comment]
Hej @einand ! Tack för informationen om Trilateration. Om du vet att det är det GSM använder (och har en källa på det), har du lust o skriva ett förslag på ny formulering runt det?

Vid triangulering använder man ofta fler än två vinklar/mätobjekt, för att även få ett mått på felmarginalen. Om du ritar in vinklar mot tre objekt, får du inte en skärningspunkt mellan linjerna, utan en liten triangel mellan de tre strecken, och storleken på triangeln är ett mått på ditt mätfel.

Ingen aning om GSM gör ngt sånt dock :)

**Comment by einand** [comment]
Nej, mobiler använder sig inte enbart av Trilateration, självklart används flera tekniker. Tex räcker det med enbart en mast för att positionerna dig med en mobil. Då dessa även vet två andra faktorer som är avgörande. Vinkel till vart du befinner dig och tid. Med detta räcker det med en station.

http://www.initsys.net/ar/about/news/item/292-a-hac
http://www-users.cs.umn.edu/~foo/research/docs/fookune_htcia.pdf
http://arrow.dit.ie/cgi/viewcontent.cgi?article=1096&context=dmccon
http://www.ipcgps.com/uploads/docs/Intro_to_Location_Technologies-1.pdf
